### PR TITLE
Climber and Wrangler Subsystem

### PR DESCRIPTION
**Pull request by CSDotBro**
This is the subsystem for the climber and wrangler (or bumper munch, depending on what you want to call it) systems. It also includes the commit that sets their correct ports.

### CODE REVIEW

File: src/main/include/subsystems/Climber.hpp [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    void SetTransverser(double speed);
+
+    void SetElevator(double speed);
+
+    void ProcessMessage(const ButtonPacket& message);
+
+private:
+    rev::CANSparkMax m_armL{frc3512::Constants::Climber::kArmPortL,
+                            rev::CANSparkMax::MotorType::kBrushless};
+    rev::CANSparkMax m_armR{kArmPortR, rev::CANSparkMax::MotorType::kBrushless};
```
Comment (calcmogul) [inline src/main/include/subsystems/Climber.hpp]: Call them m_armLeft and m_armRight. Characters are cheap.

Comment (CSDotBro) [reply src/main/include/subsystems/Climber.hpp]: Change applied.

File: src/main/include/Constants.hpp [hunk 2]
```diff
@@ -60,4 +60,57 @@ constexpr double kDpP = (wpi::math::pi * 2.0) / 512.0;
...
+    (2.0 * wpi::math::pi * kWheelRadius.to<double>()) / 2048.0;
+
+}  // namespace Drivetrain
+
+namespace Robot {
+constexpr int kDriveStick1Port = 0;
+constexpr int kDriveStick2Port = 1;
+constexpr int kAppendageStickPort = 2;
+constexpr int kAppendageStick2Port = 3;
+}  // namespace Robot
```
Comment (calcmogul) [inline src/main/include/Constants.hpp]: These already exist earlier in the file and are redundant.

File: src/main/cpp/Robot.cpp [hunk 3]
```diff
@@ -5,11 +5,18 @@
 #include <frc/DriverStation.h>
 #include <wpi/raw_ostream.h>
 
+#include <wpi/raw_ostream.h>
+
+#include <frc/DriverStation.h>
+
+#include <wpi/raw_ostream.h>
+
```
Comment (calcmogul) [inline src/main/cpp/Robot.cpp]: You have duplicate includes here. Remove one of them. Also, run wpiformat.

File: src/main/include/Constants.hpp [hunk 4]
```diff
@@ -60,4 +60,57 @@ constexpr double kDpP = (wpi::math::pi * 2.0) / 512.0;
...
+constexpr int kArmPortL = 2;
+constexpr int kArmPortR = 3;
+// Transverser Motor
+constexpr int kTransverserPort = 12;
+// Wrangler Winch Motor
+constexpr int kWinchPort = 7;
+
+}  // namespace Climber
+
+namespace Drivetrain {
```
Comment (calcmogul) [inline src/main/include/Constants.hpp]: Aren't these drivetrain constants not relevant?

**Comment by CSDotBro** [comment]
Resolved all of the comments

**Comment by CSDotBro** [comment]
Corrupted. It is now replaced with "Create the Climber Subsystem".

**Pull request closed without merging**